### PR TITLE
fix(googleads): added validation to discard event if event name is no event name is present

### PR DESCRIPTION
**Pull request by mihir-4116**
## PR Description
- This pr removes unnecessary logger error and checks if event name is present or not.if event name is not present then it will discard google ads event

## Linear task (optional)

Linear task link

## Cross Browser Tests

Please confirm you have tested for the following browsers:

- [ ] Chrome
- [ ] Firefox
- [ ] IE11

## Sanity Suite

- [ ] All sanity suite test cases pass locally

## Security

- [ ] The code changed/added as part of this pull request won't create any security issues with how the software is being used.

### CODE REVIEW

**Review by koladilip — COMMENTED** [review]
what about test cases?

**Comment by mihir-4116** [comment]
> what about test cases?

done, added test for no event name is present

**Review by koladilip — APPROVED** [review]

File: packages/analytics-js-integrations/__tests__/integrations/GoogleAds/browser.test.js [hunk 1]
```diff
@@ -76,6 +98,24 @@ describe('Scenario to test dynamic remarketing event by enabling the dynamicRema
...
 
 // New Config Test Cases
+describe('Edge scenarios', () => {
+  let googleAds;
+  beforeEach(() => {
+    googleAds = new GoogleAds(googleAdsConfigs[2], {  }, destinationInfo);
+    googleAds.init();
+    window.gtag = jest.fn();
+  });
+  test('Scenario to test no event name', () => {
```
Comment (ItsSudip) [inline packages/analytics-js-integrations/__tests__/integrations/GoogleAds/browser.test.js:108]: Can you add a test with page call also? We are using same function in page calls as well.

Comment (mihir-4116) [reply packages/analytics-js-integrations/__tests__/integrations/GoogleAds/browser.test.js]: addressed

**Review by ItsSudip — APPROVED** [review]

**Pull request merged**